docs/tsc/meetings/2019-03-11.md

```diff
@@ -0,0 +1,85 @@
+Mar 11, 2019
+
+Host: Michael Dolan
+
+Rotating Secretary: Carl Rand
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Thanh Ha - Academy Software Foundation / Linux Foundation
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+
+# **OCIO TSC Meeting Notes**
+
+* Prerequisites to moving the repository to ASWF
+    - Resolve "ext/" dependencies prior to the move.
+    - What are we going to do with the "ext/" directory?
+    - Do we maintained or replace?
```

Comment:
```suggestion
    - Do we maintain or replace?
```

Reply:
Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236